rx-groovy/src/main/asciidoc/groovy/index.adoc

```diff
@@ -103,6 +103,16 @@ transforms an `Observable<Buffer>` in json format into an object observable:
 include::unmarshaller.groovy[tags=example]
 ----
 
+== Yaml unmarshalling
```

Comment:
Jackson alternative formats are not limited to Yaml (XML for sure, I believe there are others). Could you update this title to reflect the diversity of text backends?

Reply:
Sure